pkg/kudoctl/kudoinit/prereq/namespace.go

```diff
@@ -35,9 +35,18 @@ func (o KudoNamespace) String() string {
 }
 
 func (o KudoNamespace) PreInstallVerify(client *kube.Client, result *verifier.Result) error {
+	ns, err := client.KubeClient.CoreV1().Namespaces().Get(context.TODO(), o.opts.Namespace, metav1.GetOptions{})
+
+	// If either custom or default ns is terminating, we can't install
+	if err == nil {
+		if ns.Status.Phase == v1.NamespaceTerminating {
```

Comment:
I'm not sure but wdyt about putting this check in the `health.go`? E.g. a healthy namespace has the  `NamespaceActive` phase?

Reply:
Phase can currently only be `Active` or `Terminating`, so we don't need a health check for NS (at the moment)